src/components/ProjectSearch/TextSearch.js

```diff
@@ -185,7 +185,9 @@ export default class TextSearch extends Component<Props> {
         />
       );
     } else if (status === statusType.fetching) {
```

Comment:
If you look below you will find that `} else if (this.props.query && !results.length) {` also returns the same piece of code, so it might be a good idea to join conditions with a `||`